projects/plugins/jetpack/extensions/plugins/ai-assistant-plugin/components/breve/highlight/style.scss

```diff
@@ -3,9 +3,16 @@
 [class^='jetpack-ai-breve__has-proofread-highlight'] {
 	border-bottom: 3px solid;
 	@include features-colors( ( 'border-bottom-color' ) );
+
 	&.jetpack-ai-breve__is-loading {
 		@include pulse();
 	}
+
+	&.jetpack-ai-breve__highlight-hovered {
+		transition: background-color;
+		transition-duration: 0.3s;
```

Comment:
Those transitions could be added one level above, so there is also a fade-out. Adding them here makes the fade-in work, but the background is removed abruptly on mouse leave.